template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -234,8 +247,23 @@ async def main():
     proof_root = pathlib.Path(__file__).resolve().parent
     litani = get_litani_path(proof_root)
 
+    litani_caps = get_litani_capabilities(litani)
+
     if not args.no_standalone:
         cmd = [str(litani), "init", "--project", args.project_name]
+
+        if "output_directory_flags" in litani_caps:
+            out_prefix = proof_root / "output"
+            out_symlink = out_prefix / "latest"
+            out_index = out_symlink / "html" / "index.html"
+            cmd.extend([
+                "--output-prefix", str(out_prefix),
+                "--output-symlink", str(out_symlink),
+            ])
+            print(
+                "For your convenience, the output of this run will be "
+                "symbolically linked to %s" % str(out_index))
+
```

Comment:
The problem with notes like this is that they are lost in the blizzard of output generated by run-cbmc-proofs.py

Reply:
Not much I can do about this here, unless you have suggestions, but I've also documented this in the prolog, so people can read it if they run `-h`. See "Document the presence of the `output` directory"

Reply:
But some output is crucial (where to find the report and where to find the current progress) and most output is noise.  At the very least, surrounding the crucial output by blank lines would make it stand out.

Reply:
Done